.env.example

```diff
@@ -2,3 +2,6 @@
 OPENAI_API_KEY=""
 # https://rollbar.com/signup/
 ROLLBAR_ACCESS_TOKEN=""
+
+# gpt-3.5-turbo or gpt-4
+OPENAI_GPT_MODEL=""
```

Comment:
This isn’t a secret, default it to 4.
 Also add a return at the end of the file.

Reply:
> This isn’t a secret, default it to 4. Also add a return at the end of the file.

done